### DISCUSSION TITLE
Catch "Message rejected" (opening the email so you can check it)

### DISCUSSION THREAD

**Discussion by user908348** [Extended-Thunder/send-later#322] [Q&A] [answered]
I've some scheduled emails which aren't sent and show an Alert MsgBox 

"An error occurred while sending mail. The mail server responded:  5.6.0 Message rejected, invalid message header. See rfc5322 section 2.2 [8905]. Please check the message and try again"

I can only click OK. Is there a way to "try" and "catch" the email message and open it as draft so I can identity the message? Currently it is very difficult to pinpoint the message that is causing the issue, I suspect TB and/or send later "knows" which message it is so I would like to see a "open email/message" button for example.

**Comment by jph-sendlater** [comment, marked as answer]
The process of actually sending messages is handled entirely by Thunderbird, so you'll have to file a bug with them. Sorry to not be more help, but by the time the message is being sent it's outside of the Send Later extension's hands.

**Reply by user908348** [reply]
OK, thanks - so TB should report back to an addon "success" or "failed"

**Reply by jph-sendlater** [reply]
There are a few ways that Thunderbird could handle this, and I'm not sure what the best proposal would be. I think the most likely would probably be as part of this bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1545930

In short, Thunderbird currently has no official way for extensions to send emails, but they will eventually implement one. Send Later will switch to using that function as soon as it exists. Whatever method they come up with should include some feedback mechanism to alert extensions when a message gets rejected.
